repl/evaluator.go

```diff
@@ -588,9 +607,9 @@ func (e *Evaluator) AddDeclFn(name string, params []letFunctionParam, typeHint *
 // AddOption adds an option to the basic environment.
 // Options are applied before evaluating any of the let statements.
 // Returns an error if setting the option prevents planning any of the defined let expressions.
-func (e *Evaluator) AddOption(opt Optioner) error {
+func (e *Evaluator) AddOption(opt Optioner, isYAML bool) error {
```

Comment:
nit: thoughts on splitting this into a named method to remove the potential ambiguity from boolean parameter at the call site? (ex: `AddConfigOption`, `AddYamlOption` etc?)